docs/apache-airflow/upgrade-check.rst

```diff
@@ -0,0 +1,145 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think this should be moved to a separate documentation package so that this documentation can be updated independently of Airflow. Simply put, you need to create a new directory in / docs / and then update the /docs/conf.py and /docs/build_docs.py files so that the package can be built.

Reply:
If we do this how can we link to it this from a (yet-to-be-created) `updating-to-2-0.rst` in `docs/apache-airflow/`?

Reply:
Oh -- but also, maybe slight complication: the upgrade/check code doesn't exist on master, just on v1-10-* branches.

Reply:
Documentation for airflow upgrade-check, we can publish from the master page, if we do not generate any pages automatically.

Docs about cross-reference syntax: https://github.com/apache/airflow/tree/master/docs#cross-referencing-syntax